logstash-filter-verifier.go

```diff
@@ -77,7 +81,7 @@ func runTests(logstashPath string, tests []testcase.TestCase, configPaths []stri
 		}
 
 		result, err := p.Wait()
-		if err != nil {
+		if err != nil || *logstashOutput {
 			message := fmt.Sprintf("Error running Logstash: %s.", err)
```

Comment:
`message` should only start like this if err != nil. Example of what it looks like now:

```
Running tests in client-decoration.json...
Logstash output:
Error running Logstash: %!s(<nil>).
Process output:
Sending logstash logs to /tmp/638530082.

Log:
{:timestamp=>"2017-02-02T20:03:43.426000+0000", :message=>"Reading config file", :config_file=>"/tmp/346208537/50-http-access-filter.conf.j2", :level=>:debug, :file=>"logstash/config/loader.rb", :line=>"69", :method=>"local_config"}
...
```

Perhaps just start with an empty `message` if there's no error?